packages/client/hmi-client/src/components/facets/categorical-facet.vue

```diff
@@ -232,7 +232,9 @@ export default defineComponent({
 });
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
+@import '@/styles/variables.scss';
```

Comment:
You can probably remove this now since you don't seem to be using this file? Or if this is needed for `facets-overrides.scss` you can import that file instead.

Reply:
This file is required at the moment for variables ect inside the override. This is based off of the WM design, so not sure if in the future we will be overriding completely differently